index.html

```diff
@@ -4,162 +4,89 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <meta name="author" content="Choco Li" />
-    <meta name="description" content="2 full page demonstration" />
-    <meta name="keywords" content="orchard, healthy, food" />
-    <meta name="Resource-type" content="Document" />
     <title>Orchard</title>
   </head>
-  <body>
-    <div id="fullpage">
-      <div class="section active" id="section0">
-        <section class="story">
-          <div class="content content--x-center">
-            <div class="imageBox">
-              <img
-                class="imageBox__image"
-                src="./img/crossField.png"
-                alt="a man carrying a child crossing a crop field"
-              />
+  <body class="bgBlack">
+    <div class="container">
+      <section class="row my-15 align-items-center">
+        <div class="px07 col-lg-4 col-md-6">
+          <img src="./img/crossField.png" alt="crossField" class="mw-100" />
+        </div>
+        <div class="px07 col-lg-4 col-md-6 d-flex flex-column">
+          <img src="./img/fruitInPlate.png" alt="fruit" class="mw-100 py07" />
+          <img src="./img/asparagus.png" alt="asparagus" class="mw-100 py07" />
+        </div>
+        <article class="px07 col-lg-4 col-md-12 textWhite align-self-start">
+          <h4 class="border-bottom py-3 textSpacing_04">
+            <a href="">ANSWER YOUR BODY'S NEEDS</a>
+          </h4>
```

Comment:
Just for your reference. 

I don't agree with bootstrap's way by using `<h1/>` to `<h6/>` to render different font sizes. I personally will use `<h1 class="h4 text-uppercase">` since `Answer your body's needs` looks like the title of this whole documentation. `<h2 class="h1 text-uppercase">All the latest from AEG</h2>` for the bottom module. It makes more senses for screen readers. 

However, most people keep the way you do. I simply not agree with it.

Reply:
Can't agree more.
Tags should use for document structure purpose.